tutorials/mender_gcp_ota_demo/image/update-image.sh

```diff
@@ -1,8 +1,7 @@
 #! /bin/bash
-wget https://d1b0l86ne08fsf.cloudfront.net/mender-artifact/master/mender-artifact
+wget https://d1b0l86ne08fsf.cloudfront.net/mender-artifact/2.3.0/mender-artifact
 chmod +x mender-artifact
 wget https://storage.googleapis.com/mender-gcp-ota-images/gcp-mender-demo-image-raspberrypi3.sdimg
-sudo apt install -y parted
```

Comment:
in the cloudshell linux environment, mender-artifact would not run without this. Are you saying that this should not longer be required after move from 2.3.0 to master?

In general I like to pin versions in tutorials where possible, as I've had things break. Are you pushing individual git commits to S3?  Is there a versioned release cut that has the current master improvements?

Reply:
OK, if you tested this in cloud shell we should keep this line; 2.3.0 and master should work the same in this regard - I have just never needed to do this but also never tested on the Cloud Shell Linux version. Will follow up to remove this change.

The link https://d1b0l86ne08fsf.cloudfront.net/mender-artifact/master/mender-artifact will always work but the binary there is unstable - not pinnned. It will be uploaded whenever the Mender CI triggers a build which can be several times a week.

https://d1b0l86ne08fsf.cloudfront.net/mender-artifact/2.3.0/mender-artifact is a stable tagged release, mender-artifact 2.3.0 was released with Mender version 1.6.0.